wandb/sdk/wandb_run.py

```diff
@@ -3073,8 +3075,7 @@ def _footer_log_dir_info(
 
         log_dir = settings.log_user or settings.log_internal
         if log_dir:
-            # printer = printer or get_printer(settings._jupyter)
-            log_dir = os.path.dirname(log_dir.replace(os.getcwd(), "."))
+            log_dir = os.path.dirname(log_dir)
```

Comment:
Seems like the old behavior of showing a relative path was nicer:

OLD:
wandb: Find logs at: ./wandb/run-20220427_161225-3tz4bz6z/logs
NEW:
wandb: Find logs at: /Users/jeff/work/wb/client/functional_tests/core/wandb/run-20220427_161208-2893jyzh/logs

Reply:
sure I can undo this change...